rpc/block_create.c

```diff
@@ -415,7 +415,9 @@ glusterBlockCreateRemoteAsync(blockServerDefPtr list, size_t mpath,
 
  out:
   for (i = 0; i < mpath; i++) {
-    GB_FREE(args[i].reply);
+    if (args + i) {
```

Comment:
Will it be a problem if the args == NULL ?
Will if crash ?

Reply:
Grr! makesense, let me fix it.

Reply:
Fixed now, please take look @lxbsz